teuthology/task/install.py

```diff
@@ -577,6 +529,23 @@ def remove_sources(ctx, config):
             p.spawn(remove_fn, remote, project)
 
 
+def get_package_list(config):
+    debug = config.get('debuginfo', False)
+    project = config.get('project', 'ceph')
+    yaml_path = os.path.join(os.path.dirname(__file__), 'packages.yaml')
```

Comment:
packages.yaml should exist in qa-suites, so that the construction can be /hammer/packages.yaml or similar and we dont have to overrides lots of install tasks for previous branches.